src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,55 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: ($day-size + $gap) * $col-count - $gap;
+  gap: $gap;
+  padding: $padding;
+
+  &__day {
+    display: flex;
+    height: $day-size;
+    width: $day-size;
+    border: 1px solid black;
+    background-color: #eee;
+
+    position: relative;
+    text-align: center;
+    align-items: center;
+    justify-content: center;
+    box-sizing: border-box;
+
+    &:hover {
+      cursor: pointer;
+      background-color: #ffbfcb;
+      transform: translate(0, -20px);
+      animation-duration: 0.5s;
+      transition: 0.5s;
```

Comment:
The 'transition' property should specify which properties are transitioning. Currently, it is set to '0.5s' without specifying the properties, which might lead to unexpected behavior.